flink-connector-mysql-cdc/src/main/java/com/alibaba/ververica/cdc/connectors/mysql/MySqlValidator.java

```diff
@@ -0,0 +1,135 @@
+
+    public MySqlValidator(Configuration configuration) {
+        this.configuration = configuration;
+    }
+
+    @Override
+    public void validate() {
+        try (MySqlConnection connection =
+                new MySqlConnection(new MySqlConnectionConfiguration(configuration))) {
+            if (!isVersionSatisfied(connection)) {
+                throw new TableException(
+                        "Currently Flink MySql CDC connector only supports MySql whose version is larger or equal to 5.7.");
+            }
+
+            MySqlConnectorConfig mySqlConnectorConfig = new MySqlConnectorConfig(configuration);
+            if (mySqlConnectorConfig.getSnapshotMode().shouldStream()) {
+                if (!isRowFormat(connection)) {
+                    throw new TableException(
+                            "The MySQL server is not configured to use a ROW binlog_format, which is "
+                                    + "required for this connector to work properly. Change the MySQL configuration to use a "
+                                    + "binlog_format=ROW and restart the connector.");
+                } else if (!isBinlogRowImageFull(connection)) {
+                    throw new TableException(
+                            "The MySQL server is not configured to use a FULL binlog_row_image, which is "
+                                    + "required for this connector to work properly. Change the MySQL configuration to use a "
+                                    + "binlog_row_image=FULL and restart the connector.");
+                }
+            }
+        } catch (SQLException ex) {
+            throw new TableException("Failed to validate the connection.", ex);
+        }
+    }
+
+    private boolean isVersionSatisfied(MySqlConnection connection) throws SQLException {
+        String version =
+                connection.queryAndMap("SELECT VERSION()", rs -> rs.next() ? rs.getString(1) : "");
+
+        // Only care about the major version and minor version
+        Integer[] versionNumbers =
+                Arrays.stream(version.split("\\."))
+                        .limit(2)
+                        .map(Integer::new)
+                        .toArray(Integer[]::new);
+        if (versionNumbers[0] > 5) {
+            return true;
+        } else if (versionNumbers[0] < 5) {
+            return false;
+        } else {
+            return versionNumbers[1] >= 7;
+        }
+    }
+
+    /**
+     * Adapt code from the {@link MySqlConnection} to validate the row format. Because this method
+     * is untouchable from our side and it's better to validate the requirements from the open
+     * phase.
+     */
+    private boolean isRowFormat(MySqlConnection connection) throws SQLException {
+        String mode =
+                connection.queryAndMap(
+                        "SHOW GLOBAL VARIABLES LIKE 'binlog_format'",
+                        rs -> rs.next() ? rs.getString(2) : "");
+        return "ROW".equals(mode);
```

Comment:
"ROW", "FULL" can be final static members with a meaningful name such as`BINLOG_FORMAT_ROW`,`BINLOG_ROW_IMAGE_FULL`